### PR TITLE
Shop

### PR DESCRIPTION
**Pull request by AlexMandic6**

### CODE REVIEW

**Comment by AlexMandic6** [comment]
When was making shop grid responsive, some things are hardcoded to match the design. Maybe overcomplicated this a bit. :)

File: src/js/index.js [hunk 1]
```diff
@@ -1 +1,7 @@
-import '../styles/main.scss';
\ No newline at end of file
+import '../styles/main.scss';
```
Comment (todorovicj86) [inline src/js/index.js:1]: We never import styles to js, only js-files

File: src/styles/layout/_text-banner.scss [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+
+    @include mq(phone-landscape) {
+        padding: 70px 20px;
+    }
+    @include mq(phone) {
+        padding: 35px 20px;
+    }
 }
 
 .text-banner-container {
```
Comment (todorovicj86) [inline src/styles/layout/_text-banner.scss:16]: I left the comment in your previous PR regarding the BEM. Please take a look at those comments, so I avoid add the same thing multiple times
image

**Review by todorovicj86 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 node_modules
+dist/
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply .gitignore]: Resolved.

File: package.json [hunk 4]
```diff
@@ -7,8 +7,8 @@
   "main": "index.js",
   "scripts": {
     "dev": "webpack --mode development",
-    "build": "webpack --mode production",
-    "start": "webpack-cli serve --mode development"
+    "build": "cross-env NODE_ENV=production webpack",
+    "start": "cross-env webpack-cli serve --mode development"
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply package.json:11]: Resolved

File: src/index.html [hunk 5]
```diff
@@ -7,45 +7,235 @@
...
-                <li class="nav__item">
-                    <a href="#!" class="nav__link">About Us</a>
-                    <span class="nav__span">.</span>
+            <div class="nav__logo display-none">
+                <a href="/">
+                    <img class="nav__logo-img" src="./img/logo.png" alt="Company Logo">
+                </a>
+            </div>
+            <ul class="nav__list js-nav-list">
+                <li class="nav__item nav__item-span">
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/index.html]: Only way i found to work without using additional elements in html, extra classes or JS manipulation is this selector:
:not(.nav__item-logo):not(:first-child):not(.nav__item-logo + .nav__item)::before 
;D

File: src/index.html [hunk 6]
```diff
@@ -7,45 +7,235 @@
...
+    </section>
+
+    <section class="gallery" id="work">
+        <div class="gallery__image gallery__image-big">
+            <img class="gallery__image-img" src="./img/bike1.png" alt="Photo of our bicycle">
+            <img class="gallery__image-logo abs-center" src="./img/bicycle-logo.png" alt="Bicycle company logo">
+        </div>
+        <div class="card card--right">
+            <div class="card__text">
+                <svg class="card-svg">
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/index.html]: Resolved

File: src/styles/layout/_hero.scss [hunk 7]
```diff
@@ -1,5 +1,13 @@
...
+    padding-top: calc(800 / 1600 * 100%);
+
+    height: 70vh; //seems to be working best for now
+    background: $color-hero-fallback url('../img/hero-background.png') no-repeat center;
     background-size: cover;
+
+    @include mq(phone) {
+        height: 35vh;
+        background: $color-hero-fallback url('../img/hero-background-small.png') no-repeat center;
+        background-size: cover;
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/styles/layout/_hero.scss]: Resolved

File: src/styles/layout/_slide.scss [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+    color: $color-gray-light;
+    font-family: $font-primary-italic;
+    font-weight: 400;
+}
+
+// have to use important to override Swiper's styles.
+.swiper-pagination-bullet {
+    border: 1px solid $color-gray-light;
+    width: 12px!important;
+    height: 12px!important;
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/styles/layout/_slide.scss]: Resolved

File: src/styles/layout/_text-banner.scss (hunk 2 above)
Comment (AlexMandic6) [reply src/styles/layout/_text-banner.scss]: Resolved

File: src/index.html [hunk 9]
```diff
@@ -7,45 +7,235 @@
...
+    </section>
+
+    <section class="slide">
+            <!-- Slider main container -->
+        <div class="swiper">
+            <!-- Additional required wrapper -->
+            <div class="swiper-wrapper">
+                <!-- Slides -->
+                <div class="swiper-slide">
+                    <div class="slide-container">
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/index.html]: Resolved

File: src/index.html [hunk 10]
```diff
@@ -7,45 +7,235 @@
...
+            </div>
+        </div>
+        <div class="shop-cta">
+            <button class="btn btn--black" type="button">Discover the shop</button>
         </div>
     </section>
+
+    <section class="contact" id="contact">
+        <div class="subscribe">
+            <img class="subscribe__logo" src="./img/logo.png" alt="Company Logo">
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/index.html:210]: It says in specs that subscription logo should not be clickable, its a bit strange i know.

Untitled

File: src/index.html [hunk 11]
```diff
@@ -7,45 +7,235 @@
...
+                <div class="shop-grid-image">
+                    <img class="shop-grid-img" src="./img/shop/bike-seat2.png" alt="Bike seat">
+                </div>
+                <div class="shop-grid-image">
+                    <img class="shop-grid-img" src="./img/shop/bike-bag2.png" alt="Bike bag">
+                </div>
+            </div>
+        </div>
+        <div class="shop-cta">
+            <button class="btn btn--black" type="button">Discover the shop</button>
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/index.html:204]: Resolved

File: src/styles/base/_typography.scss [hunk 12]
```diff
@@ -0,0 +1,16 @@
+@font-face {
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/styles/base/_typography.scss]: resolved

File: src/styles/components/_button.scss [hunk 13]
```diff
@@ -0,0 +1,41 @@
+.btn {
+    display: inline-block;
+    border: none;
+    padding: 1rem 2rem;
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/styles/components/_button.scss:4]: resolved

File: src/styles/layout/_contact.scss [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+    margin-top: 30px;
+    width: 530px;
+
+    @include mq(phone) {
+        width: 100%;
+        flex-direction: column;
+        align-items: center;
+    }
+
+    & .btn--green {
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/styles/layout/_contact.scss]: resolved

File: src/styles/layout/_gallery.scss [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+    }
+}
+.gallery__image {
+    position: relative;
+    @include mq(phone-landscape) {
+        grid-column: full-start / full-end;
+        order: 3;
+    }
+}
+.gallery__image.hide-on-md-screen {
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/styles/layout/_gallery.scss:23]: resolved

File: src/styles/layout/_header.scss [hunk 16]
```diff
@@ -1,40 +1,157 @@
...
+    &::after {
+        content: "";
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        background-color: $color-green;
+        right: 0;
+        bottom: -6px;
+        transition: all .2s ease-in-out;
+    }
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/styles/layout/_header.scss:135]: resolved

File: src/styles/layout/_footer.scss [hunk 17]
```diff
@@ -0,0 +1,49 @@
+.footer {
+    height: 250px;
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/styles/layout/_footer.scss:2]: deleted fixed height which was just for testing but forgot to delete :)
Added more padding to compensate for that

File: src/index.html [hunk 18]
```diff
@@ -7,45 +7,235 @@
...
+    </section>
+
+    <section class="shop" id="shop">
+        <div class="shop-wrapper">    
+            <div class="shop-grid">
+                <div class="shop-grid-image">
+                    <img class="shop-grid-img" src="./img/shop/bike-seat-small.png" alt="Bike seat">
+                </div>
+                <div class="shop-grid-image">
+                    <img class="shop-grid-img" src="./img/shop/bike-seats.png" alt="Bike seat">
```
[the remark this answers is not in the document]
Comment (AlexMandic6) [reply src/index.html]: resolved

**Review by todorovicj86 — APPROVED** [review]

**Pull request merged**